src/components/simulation/SimulationEventsPanel.svelte

```diff
@@ -78,6 +78,14 @@
       resizable: true,
       sortable: true,
     },
+    span: {
+      field: 'span',
```

Comment:
This doesn't exist on the `SimulationEvent` type. What is this supposed to contain?

Reply:
Events and Spans are separate data from the database. We are combining them into the table.